README.md

```diff
@@ -0,0 +1,274 @@
+| Windows | :heavy_check_mark: | :heavy_check_mark: (1.) | :heavy_check_mark: (1.) | :x:  (3.)          |
+| Mac     | :heavy_check_mark: | :heavy_check_mark:      | :x: (2.)                | :x:  (3.)          |
+
+1. Limiting memory and cputime on Windows is done with the library `pywin32`. There seems
+to be installation issues when instead of using `conda install <x>`, you use `pip install <x>`
+inside a conda environment, specifically only with `Python 3.8` and `Python 3.9`.
+The workaround is to instead install `pywin32` with conda, which can be done with
+`pip uninstall pywin32; conda install pywin32`.
+Please see this [issue](https://github.com/mhammond/pywin32/issues/1865) for updates.
+
+2. Mac doesn't seem to allow for limiting a processes memory. No workaround has been found
+including trying `launchctl` which seems global and ignores memory limiting. Possibly `ulimit`
+could work but needs to be tested. Using `setrlimit(RLIMIT_AS, (soft, hard))` does nothing
+and will either fail explicitly or silently, hence we advertise it is not supported.
+However, passing a memory limit on mac is still possible but may not do anything useful or
+even raise an error.
+
+3. This is something due to how multiprocessing pickling protocols work, hence `@restricted(...)` does
+not work for your Mac/Windows. Please use the `limit` method of limiting resources in this case.
+(Technically this is supported for Mac Python 3.7 though). This is likely due to the default
+`spawn` context for Windows and Mac but using other available methods on Mac also seems to not work.
+For Linux, the `fork` and `forkserver` context seems to work.
+
+
+#### Parameters
+The full list of options available with both `limit` and `@restricted` are:
+```python
+def limit(
+    func: Callable,
+    *,
+    name: str | None = None,
+    memory: int | tuple[int, str] | None = None,
+    cpu_time: int | tuple[float, str] | None = None,
+    wall_time: int | |tuple[float, str] | None = None,
+    context: str | None = None,
+    raises: bool = True,
+    warnings: bool = True,
+)
+
+# The name given to the multiprocessing.Process
+name: str | None = None
+
+# The memory limit to place. Specify the amount of bytes or (int, unit) where unit
+# can be "B", "KB", "MB" or "GB"
+memory: int | tuple[int, str] | None = None
+
+# The cpu time in seconds to limit the process to. This time is only counted while the
+# process is active.
+# Can provide in (time, units) such as (1.5, "h") to indicate one and a half hours.
+# Units available are "s", "m", "h"
+cpu_time: int | tuple[float, str] | None = None
+
+# The wall time in seconds to limit the process to
+# Can provide in (time, units) such as (1.5, "h") to indicate one and a half hours.
+# Units available are "s", "m", "h"
+wall_time: int | tuple[float, str] | None = None
+
+# Whether to throw any errors that occured in the subprocess to silently
+# throw them away. If `True` and an Error was raised, `None` will be returned.
```

Comment:
Would it be possible to still print and pass the traceback to the parent process?

Reply:
That's exactly what happens with `raises=True`, using `False` basically means you will never get an error from pynisher.

Reply:
I think this is very much in line with my comment above, so I will resolve this one.